javascript/modules/client/src/client-addressList.ts

```diff
@@ -70,173 +23,25 @@ const PLUGIN_ID = "0x1234567890123456789012345678901234567890";
  */
 export class ClientAddressList extends ClientCore
   implements IClientAddressList {
+  public methods: IClientAddressListMethods;
+  public encoding: IClientAddressListEncoding;
+  public decoding: IClientAddressListDecoding;
+  public estimation: IClientAddressListEstimation;
+
   constructor(context: ContextPlugin) {
     super(context);
+    this.methods = new IClientAddressListMethodsModule(context);
+    this.encoding = new IClientAddressListEncodingModule(context);
+    this.decoding = new IClientAddressListDecodingModule(context);
+    this.estimation = new IClientAddressListEstimationModule(context);
```

Comment:
```suggestion
    this.methods = new IClientAddressListMethods(context);
    this.encoding = new IClientAddressListEncoding(context);
    this.decoding = new IClientAddressListDecoding(context);
    this.estimation = new IClientAddressListEstimation(context);
```

Reply:
I was just following the naming that we used in the `ipfs`, `web3` and `graphql` modules which is `IpfsModule` `Web3Module` and `GraphqlModule` respectively. I would definitely drop the `I` because it is not an interface and does not make sense,`IClientAddressListMethodsModule` => `ClientAddressListMethodsModule` and about the module at the end if we remove it I would remove it everywhere `class Web3Module` => `class Web3` otherwise we wouldn't be consistent with the naming

Reply:
Well, we should use interfaces for typing class methods, otherwise we're going to have issues if we type with classes.
But as per class names, they should definitely be without an `I`. 

Regarding the `Module` suffix, it is good that we have consistency, but GraphQL/IPFS/Web3 are a different kind of concept than "Encoding", etc. These are groups of functions of the same type, I don't think `Module` helps understanding much here, even more given the fact that this is internal code that external users should not be working with.